### PR TITLE
Lth/mug footer

### PR DESCRIPTION
**Pull request by lpandath**
Updated the footer template (footer_mug.html)
Added configuration keys for enabling/disabling the resource overview 
Updated configuration structures

### CODE REVIEW

File: invenio_override/config.py [hunk 1]
```diff
@@ -69,13 +95,13 @@
 THEME_HEADER_LOGIN_TEMPLATE = "invenio_override/accounts/header_login.html"
 """login page header"""
 
-THEME_FOOTER_TEMPLATE = "invenio_override/footer.html"
+THEME_FOOTER_TEMPLATE = "invenio_override/footer_mug.html"
```
Comment (mb-wali) [inline invenio_override/config.py:98]:
```suggestion
THEME_FOOTER_TEMPLATE = "invenio_override/footerhtml"
```
use the default for this repo, as we discussed

File: invenio_override/templates/invenio_override/contact_us.html [hunk 2]
```diff
@@ -18,11 +18,14 @@ <h4>{{ _("Need help?") }}</h4>
   {%- endif %}
 
   <div style="padding-bottom: 10px;">
+    {%- if config.SHOW_RIGHT_CONTACT_EMAIL %}
```
Comment (mb-wali) [inline invenio_override/templates/invenio_override/contact_us.html:21]:
```suggestion
    {%- if config.SHOW_RIGHT_CONTACT_EMAIL %}
  <div style="padding-bottom: 10px;">
```
if the contact email is false, we dont need the <div> allocating space in body

**Review by mb-wali — COMMENTED** [review]
left some comments.

File: invenio_override/config.py (hunk 1 above)
Comment (lpandath) [reply invenio_override/config.py:98]: yap, sorry i forgot to revert while testing it! Ill revert this.

**Pull request merged**